### PR TITLE
[tests-only][not-for-merge] check PR for core 39666

### PR DESCRIPTION
**Pull request by kiranparajuli589**
### Description
- just a test pr for https://github.com/owncloud/core/pull/39666

### Tested using
- https://github.com/owncloud/ocis/pull/2939

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiFederationToShares2-10.8.0-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/113/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiWebdavUpload2-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/99/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiWebdavProperties1-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/96/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiWebdavLocks2-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/89/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIRenameFiles-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/136/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiShareUpdateToRoot-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/79/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiWebdavOperations-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/94/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiProxySmoke-8-6-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/176/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>cliAppManagement-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/119/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiFederationToRoot1-10.8.0-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/104/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiOnSqlite-sqlite-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/179/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUILogin-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/131/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUISharingExt2-git-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/159/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIFileActionsMenu-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34231/155/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiFavorites-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34253/52/1

File: tests/acceptance/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -3610,6 +3611,20 @@ public function clearFileLocks():void {
...
+	/**
+	 * @AfterScenario
+	 *
+	 * clear space id reference
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function clearSpaceId():void {
+		if (WebDavHelper::$spacesIdRef) {
```
Comment (kiranparajuli589) [inline tests/acceptance/features/bootstrap/FeatureContext.php:3623]:
```suggestion
		if (\count(WebDavHelper::$spacesIdRef) > 0) {
```

**Pull request closed without merging**